dockerfile

```diff
@@ -0,0 +1,23 @@
+FROM python:3.7-slim-buster as base
```

Comment:
Conventionally the Dockerfile is named with a capital D.

Reply:
Will fix in next commit